.bazelrc

```diff
@@ -8,3 +8,6 @@ build:rustfmt --output_groups=+rustfmt_checks
 # Enable clippy for all targets in the workspace
 build:clippy --aspects=//rust:defs.bzl%rust_clippy_aspect
 build:clippy --output_groups=+clippy_checks
+
+# https://bazel.googlesource.com/bazel/+/master/site/docs/windows.md#enable-symlink-support
```

Comment:
while doc suggests to enable runfiles (`build --enable_runfiles`), enabling it makes `process_wrapper/std_err` test fails somehow.

Reply:
What is the failure you see when enabling this?

Reply:
I ran some tests and it seems to test no problem for me. Though I used
```suggestion
# https://bazel.googlesource.com/bazel/+/master/site/docs/windows.md#enable-symlink-support
startup --windows_enable_symlinks
```
Per the recommendation from https://docs.bazel.build/versions/main/windows.html

Does that help here?

Reply:
weirdly I tried again and couldn't repro the error. maybe one off from the system I used?

Reply:
I think these test failures are by enabling runfiles, which is outside of test_env/.

```
(03:37:10) FAIL: //test/process_wrapper:stdout_test (see C:/b/h3ziskxk/execroot/rules_rust/bazel-out/x64_windows-fastbuild/testlogs/test/process_wrapper/stdout_test/test_attempts/attempt_1.log)
```